responses/21-target-expense.md

```diff
@@ -3,12 +3,12 @@ In general, if building part of a pipeline is "expensive" (i.e., takes more than
 Additional reasons to create a target include:
```

Comment:
Yes to the YAML file still being there. That is noted in my first bullet in this PR description as something that needs to be done. Since this is a real example of a scipiper workflow, I wasn't sure if we should try to pick a **targets** example? We should also choose some different examples to link to for existing pipelines so that we can share real **targets** examples.

Reply:
Oh, yep, I see your PR bullet and had forgotten about it because it took me so long to finish my review =). I'm also not sure about what example to use. I had actually assumed that this remake.yml was already a simplification of the original, but I hadn't checked. I do think that rewriting the same example in targets would get the intended points across.

For targets examples, we have https://code.usgs.gov/wma/wp/res-temperature-process-models and https://code.usgs.gov/wma/wp/forecasted-met-drivers, but I'm worried that both of these are still in flux and may not uniformly use the best practices we're pushing in this course. Rather than trying to work it in this time, including a real DS targets example could be a goal for an FY22 update of these courses.

Reply:
Agree that these might be in flux. I think it would still be nice for newbies to see how complex they can be but just give a bit of a caveat about how they are in flux and developed early on in our collective **targets** experience.

I made an issue to revisit the examples later https://github.com/USGS-R/ds-pipelines-targets-2-course/issues/4

Reply:
Although, I just realized that those are only accessible internally, so maybe they aren't the best ones to use.

Reply:
I took a stab at writing a short paragraph motivating this issue. Let me know what you think. https://github.com/USGS-R/ds-pipelines-targets-2-course/pull/2/commits/b5307cfb945f01a23945515427f04b66d02caded